android/app/src/main/java/edu/gatech/cs2340/gtrational/rational/model/User.java

```diff
@@ -82,17 +152,25 @@ public boolean setSessionId(String s) {
         return false;
     }
 
+    /**
+     * Ends the User's session.
+     */
     public void endSession() {
         sessionId = null;
     }
 
+    /**
+     * Logs a User into the system and sets a session ID.
+     * @param username the username of the User to login.
+     * @param password the password of the User to login.
+     * @return whether the operation was successful.
+     */
     public boolean login(String username, String password) {
         if (username == null || password == null) {
             return false;
         }
         if (username.equals(this.username) && password.equals(this.password) && !isLocked) {
-            //log in
-            //sessionId = database.getSessionID();
+            this.sessionId = FakeBackend.login(username, password);
```

Comment:
We shouldn't be using FakeBackend in our code but rather WebAPI since we're going to delete FakeBackend eventually, but WebAPI will live on forever